components/service/glean/src/main/java/mozilla/components/service/glean/UuidMetricType.kt

```diff
@@ -53,6 +53,7 @@ data class UuidMetricType(
                 stores = sendInPings,
                 category = category,
                 name = name,
+                lifetime = lifetime,
```

Comment:
Random thought: Would it be better to pass `this as CommonMetricData` to the storage engine rather than passing all of this common metadata independently.  Should require fewer updates to all of the storage engines as we iterate on this stuff.

Reply:
That is a great idea! Let's take this to a separate bug though, and defer after MVP (I filed bug 1506061).